app/controllers/families_controller.rb

```diff
@@ -0,0 +1,30 @@
+class FamiliesController < ApplicationController
+
+  def new
+    @family = Family.new
+  end
+
+  def create
+    @family = Family.create(family_params)
+    respond_with @family
+  end
+
+  def index
+    @families = Family.all
+  end
```

Comment:
A suggestion to move `index` action to be the first in the controller, because convention

Reply:
Done! Thanks I didn't know about that convention.